src/yaml-language-support/yaml-support.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from 'vscode';
+
+import { parse, findNodeAtPosition, util as yamlUtil } from 'node-yaml-parser';
+
+export interface YamlCache {
+    // the document model list represents the yaml text
```

Comment:
I don't understand this comment.  Does this represent a single YAML document, or a list of YAML documents?  What is meant by 'the document model list'?

Reply:
a yaml file will be able to contain multiple yaml documents separated by ---, so a list of documents is used to represents the yaml file ,I will update the comments